seed_data/challenges/computer-science.json

```diff
@@ -113,8 +130,12 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende sobre la Lógica Booleana",
+      "descriptionEs": [
+        "Ahora trabajaremos más ejercicios de tablas y aprenderemos sobre la lógica booleana.",
+        "También aprenderemos la diferencia entre data digital y data análoga.",
```

Comment:
Data should be translated as "datos", so this paragraph should be "También aprenderemos la diferencia entre datos digitales y datos análogos."